espree.js

```diff
@@ -368,8 +534,14 @@ function parse(code, options) {
         if (extra.loc) {
             acornOptions.locations = true;
         }
-    }
 
+        if (extra.ecmaFeatures.jsx) {
+            if (extra.ecmaFeatures.spread !== false) {
```

Comment:
I did that for now just to make spread tests work; this should be added to other "TODO"s for separate analysis of options.